components/calendar/src/iso.rs

```diff
@@ -383,22 +384,25 @@ impl Iso {
     // Lisp code reference: https://github.com/EdReingold/calendar-code2/blob/1ee51ecfaae6f856b0d7de3e36e9042100b4f424/calendar.l#L1167-L1189
     pub(crate) fn fixed_from_iso(date: IsoDateInner) -> i32 {
         // Calculate days per year
-        let mut fixed: i32 = EPOCH - 1 + 365 * (date.0.year - 1);
+        let mut fixed: i32 = (EPOCH - 1 + 365).saturating_mul(date.0.year.saturating_sub(1));
```

Comment:
issue: I don't think this correctly preserves the math, the `EPOCH - 1` should be outside the whole thing

Reply:
bit surprised this bug does not cause test failures

(we should ensure there are tests that would fail if htis code is broken)